### DISCUSSION TITLE
"Beat score" mod

### DISCUSSION THREAD

**Discussion by ninjastar1123** [ppy/osu#13069] [Ideas]
This feature would be a new mod added to the game. This mod would be specifically designed for the sole purpose of beating scores. When this mod is turned on, the player is trying to beat a previous score. If the player is at a certain part of the map and it’s impossible to beat their previous score, the game would automatically end. I think this would be a great feature for people that are going for fcs but don’t want to put on sd just in case they for example slider break and lose all of their progress on the map. This mod definitely would interfere with combo and sliders but considering with how osu! lazer caps out score at 1,000,000, this should make it easier to work with then the standard osu! scoring. 

This would go in the normal mod hub and the logo could be some kind of score beating logo. The score multiplier would be 0 considering that this is just a mod helping a player trying to better their score.

**Comment by frenzibyte** [comment]
I would make this a general setting in mod selection overlay rather than a whole mod.

**Comment by bdach** [comment]
Might be hard to implement perfectly due to some hitobjects providing extra score (standard spinners, taiko shakers), but I guess you could get close enough without them and even then you could try going for a cap of how much score you can get from those? Definitely an interesting idea as far as I'm concerned.
